manager/db.go

```diff
@@ -60,128 +66,249 @@ func (m *Manager) lookupCatalog(environmentId, name string) (model.Catalog, erro
 }
 
 func (m *Manager) updateDb(catalog model.Catalog, templates []model.Template, newCommit string) error {
```

Comment:
This function is huge. It would help the readability if you extract pieces of functionality into their own functions so that you could look at updateDb and get an idea of the overall logic and flow. This is over-simplifying and taking it to the extreme but it might look something like this:
```
udpateDb() {
  dbTemplates, dbTemplateCommits, dbTemplateLookups := getCatalogTemplates()
  createNewTemplates()
  deleteTemplates()
  syncLables()
  syncCategories()
}
```